equinix/resource_fabric_port.go

```diff
@@ -208,46 +208,6 @@ func FabricPortResourceSchema() map[string]*schema.Schema {
 	}
 }
 
-func readFabricPortResourceSchemaUpdated() map[string]*schema.Schema {
-	sch := FabricPortResourceSchema()
-	sch["uuid"].Computed = true
-	sch["uuid"].Optional = false
-	sch["uuid"].Required = false
-	return sch
-}
-
-func readFabricPortsResponseSchema() map[string]*schema.Schema {
```

Comment:
It looks like all of this stuff was moved into the port data source file; does that need to happen as part of introducing a new cloud routers data source?

Reply:
Nope, too much with one change. Not necessary.

Was going towards the same destination with all data source code belonging together but it should be with a different PR.